File: QViz.NET/Periscope.QViz/JSON/GUIResult.cs
using System.Collections.Generic;

namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for GUI Test Result Details
	/// </summary>
	public class GUIResult : TestResult
	{
		/// <summary>
		/// List of Test Step Result Objects
		/// </summary>
		public IList<TestStepResult> testStepResults;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public GUIResult() : base()
		{
			this.testStepResults = new List<TestStepResult>();
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/GUIResult.java
package com.periscope.qviz.json;

import java.util.ArrayList;
import java.util.List;

/**
 * JSON Model for GUI Test Result Details
 */
public class GUIResult extends TestResult {

	/**
	 * List of Test Step Result Objects
	 */
	public List<TestStepResult> testStepResults;

	/**
	 * Default Constructor
	 */
	public GUIResult() {
		super();
		this.testStepResults = new ArrayList<>();
	}

}
